src/pal/src/shmemory/shmemory.cpp

```diff
@@ -455,14 +455,14 @@ BOOL SHMInitialize(void)
     init_waste();
     
 #ifndef CORECLR
```

Comment:
But then please revert all your changes in the #ifndef block. I have thought the removal would be easier for you.